src/main/generic/consensus/base/account/tree/AccountsTreeNode.js

```diff
@@ -273,6 +273,26 @@ class AccountsTreeNode {
         return this._hash;
     }
 
+    /**
+     * Returns the number of children this node has
+     * @param {boolean} includeBranchNodes
+     * @returns {number}
+     */
+    numberOfChildren(includeBranchNodes = true) {
+        let childrenToCount;
+
+        if (includeBranchNodes) {
+            childrenToCount = this._childrenSuffixes || [];
+        } else {
+            // Only terminal nodes have suffixes that equal 40 hex numbers when added to the parent's prefix
+            childrenToCount = this._childrenSuffixes.filter(suffix => (this._prefix + suffix).length == 40);
```

Comment:
Use `===` to prevent type coercion.
Also, there is a constant `Address.HEX_SIZE` which should be used instead of the hardcoded literal.